.travis.yml

```diff
@@ -60,3 +57,8 @@ cache:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+  - git fetch --all && git config --global user.email "-" && git config --global user.name "-"
```

Comment:
`git fetch --all && git config --global user.email "-" && git config --global user.name "-"` the previous netlify build executes this, but I'm not sure what does it do? May I clarify with you guys @eugenepeh @yong24s

Reply:
Agreed; will be good to determine if this line is necessary.

Reply:
`git fetch --all` is necessary because we need it to have the full git history to get the version correctly. The git config one I am not sure either.

Reply:
@eugenepeh can I seek for your clarification on this?